gammapy/image/tests/test_lists.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from gammapy.datasets import FermiGalacticCenter
+import numpy as np
+
+
```

Comment:
You will have to add [requires_data](http://docs.gammapy.org/en/latest/api/gammapy.utils.testing.requires_data.html) here so that the test is skipped on builds that don't have access to test data.